src/client/auth/oidc.rs

```diff
@@ -69,33 +143,77 @@ pub struct IdpServerInfo {
     pub request_scopes: Option<Vec<String>>,
 }
 
+#[derive(Debug)]
+pub struct CallbackContext {
+    pub timeout_seconds: Option<Instant>,
+    pub version: i32,
+    pub refresh_token: Option<String>,
+    pub idp_info: Option<IdpServerInfo>,
+}
+
 #[derive(TypedBuilder)]
 #[builder(field_defaults(setter(into)))]
 #[non_exhaustive]
 pub struct IdpServerResponse {
     pub access_token: String,
-    pub expires: Option<Instant>,
+    pub expires_in_seconds: Option<Instant>,
```

Comment:
I'd expect something named `expires_in_seconds` to be containing a duration of some kind, not a moment in time; can you give more info on why this was renamed?

Reply:
This was the name I aped from the python implementation. I'm fine with going back to `expires`!